### PR TITLE
reconciles merge issues with DRYfooter

### PR DESCRIPTION
**Pull request by avophile**
This is a combination of pull requests #65 and #68.

### CODE REVIEW

File: views/guide.lavender [hunk 1]
```diff
@@ -57,99 +49,25 @@ block content
...
-              // or if you need to pass data into the template (probably the case)
-
-              $output = lavender::view('some_template')->compile(array(
-                'data'      => 'some data that the template will use',
-                'more_data' => 'moar data',
-              ));
-        h2#langref Language Reference
-          h3#html html
-          p
-            | This:
```
Comment (sugarPhlox) [inline views/guide.lavender:136]: Deleted this line by accident

File: views/download.lavender [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+-current_page = "Download"
+
+block title
+  | Download - Lavender Templating Engine for PHP
+
+block content
+  .container
+    div.col-md-3
+      
+    div.col-md-8
```
Comment (zlincoln) [inline views/download.lavender]: remove div.col-md-3, add class .col-md-offset-3 to div.col-md-8

**Comment by zlincoln** [comment]
:cake:

**Comment by sugarPhlox** [comment]
:cake:

**Pull request merged**